### PR TITLE
Kmr repair random stuff

### PR DESCRIPTION
**Pull request by complikatyed**
1. Used Sandy's DOM outputting strategy to show stats messages for both hero and enemy in separately styled divs.  We will want to refine the initial stats message -- I got sort of silly with it -- and changes to the styling will help it look more polished.
2. Repaired 'surprise me' class choosing mechanism for hero creation and created option for a random weapon choice as well (which is how I discovered that generateClass was broken for hero).
3. Deleted unnecessary generateClass definitions in the enemy prototypes.

### CODE REVIEW

File: javascripts/create.js [hunk 1]
```diff
@@ -65,56 +76,67 @@ var Gauntlet = (function(originalGauntlet){
...
+      case "Dagger":
+        y = new Gauntlet.Armory.Dagger();
         break;
       case "BroadSword":
-        var y = new Gauntlet.Armory.BroadSword();
+        y = new Gauntlet.Armory.BroadSword();
         break;
       case "WarAxe":
-        var y = new Gauntlet.Armory.WarAxe();
+        y = new Gauntlet.Armory.WarAxe();
```
Comment (stevebrownlee) [inline javascripts/create.js:96]: Notice how the string in each of these case statements is the exact same string of characters as the instance you are creating? Even the case matches. If you think about it, that makes this entire switch statement obsolete and unnecessary.

**Hint:** Square bracket notation for object properties.

**Comment by davidcramb** [comment]
👍

**Comment by SandraJones** [comment]
:+

**Comment by SandraJones** [comment]
:+1

**Comment by SandraJones** [comment]
:+

**Comment by SandraJones** [comment]
Thumbs Up!

**Pull request merged**